ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<p>Can you cleanup created artifacts like whatever this creates?</p><p> </p><pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; aiplatform.start_run(run=f"auto-mpg-local-run-{i}")
&nbsp; &nbsp; aiplatform.log_params(params)
&nbsp; &nbsp; model, history = train(
&nbsp; &nbsp; &nbsp; &nbsp; normed_train_data,
&nbsp; &nbsp; &nbsp; &nbsp; train_labels,
&nbsp; &nbsp; &nbsp; &nbsp; num_units=params["num_units"],
&nbsp; &nbsp; &nbsp; &nbsp; activation="relu",
&nbsp; &nbsp; &nbsp; &nbsp; epochs=params["epochs"],
&nbsp; &nbsp; &nbsp; &nbsp; dropout_rate=params["dropout_rate"],
&nbsp; &nbsp; )
&nbsp; &nbsp; aiplatform.log_metrics(
&nbsp; &nbsp; &nbsp; &nbsp; {metric: values[-1] for metric, values in history.history.items()}
&nbsp; &nbsp; )
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
Discussed offline, right now, we do not have function to clean up specific artifacts after creation. We are working on enable that capability and will add those into the notebook once supported